tests/test_e2e.py

```diff
@@ -0,0 +1,71 @@
+import random
+import string
+import time
+
+from dewy.chunks.models import RetrieveRequest, RetrieveResponse
+from dewy.documents.models import AddDocumentRequest
+
+SKELETON_OF_THOUGHT_PDF = "https://arxiv.org/pdf/2307.15337.pdf"
+
+
+async def create_collection(client, text_embedding_model: str) -> int:
+    name = "".join(random.choices(string.ascii_lowercase, k=5))
+    create_response = await client.put("/api/collections/", json={"name": name})
```

Comment:
It would be nice if we could compile the python client and use it here.

Reply:
I haven't looked at the compiled swagger client. I feel like that should be deferred from this PR at least -- should we create a ticket for it?